README.md

```diff
@@ -1 +1,54 @@
-# trace-event-logger
\ No newline at end of file
+# Trace Event Logger
+
+This logger is based on JUL to allow fast JSON traces to be written to disk. It is not lockless or nanosecond precise, but is fast and simple to use and configure.
+
+A Snapshot taker is provided in order to write slow transactions onto disk.
+
+An asynchronous writer is available for putting the IO bottleneck on another thread. It is typically much faster than the FileHandler from JUL.
+
+This is a logger helper designed to facilitate entry-exit analysis. The events generated by this logger are saved in a JSON-like message format, providing extra information associated with each event. The typical event types include:
+
+- **Durations:**
+  - **B:** Begin
+  - **E:** End
+  - **X:** Complete (event with a duration field)
+  - **i:** Instant / Info
+
+- **Asynchronous Nested Messages:**
+  - **b:** Nested Begin
+  - **n:** Nested Info
+  - **e:** Nested End
+
+- **Flows:**
+  - **s:** Flow Begin
+  - **t:** Flow Step (Info)
+  - **f:** Flow End
+
+- **Object Tracking:**
+  - **N:** Object Created
+  - **D:** Object Destroyed
+
+- **Mark Events:**
+  - **R:** Marker Event
+
+- **Counter Events:**
+  - **C:** Counter Event
+
+## Usage
+
```

Comment:
Reading the usage I think it's not clear for users how to use it. How to create a log file should be described as well (using logging.properties or programatically). Then it would be to explain also the different file handlers and how to use them when to one over the other. 

Should there be a link on a trace viewer (e.g. Trace Compass) that is able to visualize the data. Especially the scope logs.

Reply:
Added, great idea!